var/spack/repos/builtin/packages/silo/package.py

```diff
@@ -129,6 +130,8 @@ def flag_handler(self, name, flags):
             if spec.satisfies("%oneapi"):
                 flags.append("-Wno-error=int")
                 flags.append("-Wno-error=int-conversion")
+            if "+python" in spec:
+                flags.append("-I {0}".format(spec["python"].headers.directories[0]))
```

Comment:
```suggestion
                flags.append(f"-I {spec['python'].headers.directories[0]}")
```
F-strings preferred in new code.